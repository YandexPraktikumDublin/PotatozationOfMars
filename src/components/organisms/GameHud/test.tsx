import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { GameHud } from '.'

describe('<GameHud />', () => {
  it('should renders correct <GameHud />', () => {
    const string = ''
    const number = 0
    const wrapper = shallow(
      <GameHud title={string} imageSrc={string} value={number} />
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
