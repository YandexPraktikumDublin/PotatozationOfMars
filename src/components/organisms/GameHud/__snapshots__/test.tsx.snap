// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameHud /> should renders correct <GameHud /> 1`] = `
<div
  className="flex"
>
  <img
    alt=""
    className="w-8 h-8 mr-2"
    src=""
  />
  <div
    className="text-white text-2xl"
  >
    x
    0
  </div>
</div>
`;
